doc/api/report.md

```diff
@@ -395,6 +395,21 @@ is provided below for reference.
     "/lib64/libpthread.so.0",
     "/lib64/libc.so.6",
     "/lib64/ld-linux-x86-64.so.2"
+  ],
+  "requests": [
```

Comment:
The term "requests" is ambiguous here.  Maybe we can merge these items into the section "libuv".